include/gatekeeper_gt.h

```diff
@@ -189,11 +189,15 @@ struct gt_config {
 
 	/* The gt instances. */
 	struct gt_instance *instances;
+
+	/* Lua bytecode generated for incrementally updating GT lua states. */
+	char               lua_bytecode[128];
```

Comment:
128 bytes is too little for this bytecode. Drop this field.

Allocate memory for each instance with `rte_malloc_socket()`, copy the bytecode into those memory blocks, and each GT instance will free the memory. The preference for `rte_malloc_socket()` is to match the socket of the destination GT instance.